src/Artsy/Relay/renderWithLoadProgress.tsx

```diff
@@ -38,7 +38,7 @@ const handleError = error => {
       console.error(`Metaphysics Error data:`, data)
       logger.error(data)
     } catch (e) {
-      logger.error('Metaphysics Error could not be parsed.')
+      logger.error("Metaphysics Error could not be parsed.", error)
```

Comment:
@jonallured - This should return the error from the `catch(e)`, not from up above, as that won't give us the correct  infor as to why the try branch failed.

Reply:
Ugh, nice catch! I will open a PR for this. 👍

Reply:
Good catch, thanks @damassi !